web/pdf_layer_viewer.js

```diff
@@ -13,6 +13,11 @@
  * limitations under the License.
  */
 
+/** @typedef {import("./event_utils.js").EventBus} EventBus */
+// eslint-disable-next-line max-len
+/** @typedef {import("../src/optional_content_config.js").OptionalContentConfig} OptionalContentConfig */
+/** @typedef {import("../src/document.js").PDFDocument} PDFDocument */
```

Comment:
That file doesn't even exist, hence this cannot possibly work.
```suggestion
/** @typedef {import("../src/display/api").PDFDocumentProxy} PDFDocumentProxy
```